Docs: Add links to 7.3 patch release notes (#30292)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-3.md

A new standard field [color scheme]({{< relref "../panels/field-options/standard-field-options.md#color-scheme" >}}) option has been added. This new option will provide a unified way for all new panels to specify how colors should be assigned.

> - **Single color**: Specify a single color, useful in an override rule.
> - **From thresholds**: Informs Grafana to take the color from the matching threshold.
> - **Classic palette**: Grafana will assign color by looking up a color in a palette by series index. Useful for Graphs and pie charts and other categorical data visualizations.
> - **Green-Yellow-Red (by value)**: This is a continuous color scheme where Grafana will interpolate a color based on the value being displayed and the field  min & max values.
> - **Blue-Yellow-Red (by value)**: Same as above but different colors.
> - **Blues (by value)**: Same as above but color scheme go from panel background to blue.

Reviewer: ```suggestion
- **Single color**: Specifies a single color. Useful in an override rule.
- **From thresholds**: Informs Grafana to take color from the matching threshold.
- **Classic palette**: Assigns a color by looking up a color in a palette by series index. Useful for Graphs and pie charts, and other categorical data visualizations in Grafana.
- **Green-Yellow-Red (by value)**: A continuous color scheme where Grafana will interpolate a color based on the value assigned to the green, yellow, and red components. The value must be within the min & max limits.
- **Blue-Yellow-Red (by value)**: Same as above but different colors.
- **Blues (by value)**: Same as above but color scheme go from panel background to blue.
```

Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) for a complete list of new features, changes, and bug fixes.

## Patch release notes

- [Grafana 7.3.0 release notes]({{< relref "../release-notes/release-notes-7-3-0.md" >}})
- [Grafana 7.3.1 release notes]({{< relref "../release-notes/release-notes-7-3-1.md" >}})
- [Grafana 7.3.2 release notes]({{< relref "../release-notes/release-notes-7-3-2.md" >}})
- [Grafana 7.3.3 release notes]({{< relref "../release-notes/release-notes-7-3-3.md" >}})
- [Grafana 7.3.4 release notes]({{< relref "../release-notes/release-notes-7-3-4.md" >}})
- [Grafana 7.3.5 release notes]({{< relref "../release-notes/release-notes-7-3-5.md" >}})
- [Grafana 7.3.6 release notes]({{< relref "../release-notes/release-notes-7-3-6.md" >}})

> - [Grafana 7.3.7 release notes]({{< relref "../release-notes/release-notes-7-3-7.md" >}})

Reviewer 2: Can we link to the root release notes page instead, patch releases & release notes are heavily automated and this would get quickly outdated (ie missing new patch releases)

Author: That is a good point, but OTOH, this saves users a click and scanning the page with the whole list of patch releases to find the one they need. The writers will be responsible for maintaining this, and odds are decent that the community will help out as well if we are slow. Let's try it this way for a while. We can change if it turns out to be unfeasible.

Author: @achatterjee-grafana, what are your thoughts?

Reviewer: @oddlittlebird, less clicking is a always better user experience. But if the links are out of date quickly, it can become a maintenance issue. Let us see how this goes, as you said we can make adjustments as needed.
